packages/retail-ui-extensions/src/component-sets/Basic.ts

```diff
@@ -17,6 +18,7 @@ import {
   Text,
   Section,
   TextField,
+  TimePicker,
```

Comment:
Sorry could you add the PinPad component here while this PR is still up? Looks like I missed it in my PR 😅